src/lib/execution_plan.rs

```diff
@@ -143,6 +145,44 @@ fn update_member_path(member: &str) -> String {
     member_path
 }
 
+pub(crate) fn crate_target_triple(path: impl Into<PathBuf>) -> Option<String> {
```

Comment:
i think we can extract this logic to the crate_info? its under the env folder.

Reply:
Done.